src/test/java/com/facebook/presto/pravega/CodecSupplier.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This is missing a copyright, is it imported code from Presto? If so, we should indicate.

Reply:
yes, it is imported from presto.  had asked a few times what to do here w.r.t headers in this case and still was unsure.  i can add something